catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -94,6 +98,22 @@
 
     private FilterBuilder filterBuilder;
 
+    static {
+        Bundle bundle = FrameworkUtil.getBundle(Historian.class);
+        BundleContext context = bundle == null ? null : bundle.getBundleContext();
+        if (bundle == null || context == null) {
+            LOGGER.error("Could not get bundle to register history metacard types!");
+        } else {
+            context.registerService(MetacardType.class,
```

Comment:
If the History MetacardType lives in catalog-core-versioning, should they be registered there instead?